scripts/word_freq_table.js

```diff
@@ -0,0 +1,50 @@
+window.addEventListener("DOMContentLoaded", function () {
+    function checkAllVisible() {
+        const checkBoxes = document.getElementsByClassName(`cb_${this.dataset.instance}`);
+        for (const checkBox of checkBoxes) {
+            const tBody = checkBox.closest("tbody");
+            if (getComputedStyle(tBody).display !== "none") {
```

Comment:
do we need the computed style or could we use presence of some class or data- attribute instead? From memory computing styles blocks a lot of parallel execution in the browser since it needs css and javascripts to execute fully.

Reply:
I'll look into it.